tableau_twitter.py

```diff
@@ -1,254 +1 @@
-# -*- coding: utf-8 -*-
 
-from tweepy import Stream
-from tweepy import OAuthHandler
-from tweepy import API
-from tweepy.streaming import StreamListener
-import csv
-import json
-from keys import keys
-import os
-import dataextract as tde
-import re
-from datetime import datetime
-from textblob import TextBlob
-import time
-
-# put your keys here. Get them by registering an app on dev.twitter.com
-SCREEN_NAME = keys['screen_name']
-CONSUMER_KEY = keys['consumer_key']
-CONSUMER_SECRET = keys['consumer_secret']
-ACCESS_TOKEN = keys['access_token']
-ACCESS_TOKEN_SECRET = keys['access_token_secret']
-
-# this is the term we will search for on twitter
-TRACK_TERM = "#win"
-
-# some ugly global variables
-tweet_count = 0
-file_number = 0
-current_file = None
-
-WORKING_DIRECTORY = os.getcwd()
-
-def record_tweet(status, file_number):
-    # this function unpacks a tweet in JSON form and writes it to a CSV file.
-    # it also calculates the sentiment of the text of the tweet.
-
-    tweet = json.loads(status)
-
-    if 'limit' in tweet:
-        print "hit limit:", tweet
-
-    data_to_write = {}
-
-    try:
-        data_to_write["tweet_text"] = tweet['text']
-
-        # source comes as a long string: "<a href=""http://twitter.com"" rel=""nofollow"">Twitter Web Client</a>
-        data_to_write["source"] = re.findall(r">(.*)<", tweet['source'])[0]
-
-        data_to_write["user"] = tweet['user']['screen_name']
-        data_to_write["lang"] = tweet['lang']
-        data_to_write["created_at"] = tweet['created_at']
-
-        if tweet['geo'] != None:
-            lat, lon = str(tweet['geo']['coordinates'])[1:-1].split(', ')
-            data_to_write["latitude"] = lat
-            data_to_write["longitude"] = lon
```

Comment:
I don't like if else statements if we can avoid them. why not initialize data_to_write's values to be blank by default?